components/src/core/ChannelValue/ChannelValue.tsx

```diff
@@ -87,6 +105,19 @@ const ChannelValueRender: React.ForwardRefRenderFunction<unknown, ChannelValuePr
         ...otherSpanProps
     } = props;
     const defaultClasses = useStyles(props);
+    const prefixUnitWhitelist = ['$'];
+    const suffixUnitWhitelist = ['%', '℉', '°F', '℃', '°C', '°'];
+    const applyPrefix = useCallback((): boolean => {
+        if ((!prefixUnitWhitelist.includes(units) && unitSpace !== 'hide') || unitSpace === 'show') {
+            return true;
+        }
+    }, [prefix, units, unitSpace]);
+
+    const applySuffix = useCallback((): boolean => {
+        if ((!suffixUnitWhitelist.includes(units) && unitSpace !== 'hide') || unitSpace === 'show') {
+            return true;
+        }
+    }, [prefix, units, unitSpace]);
```

Comment:
Prefix doesn't seem to be used in this callback so it might not be required in deps arr.  My React is rusty so please correct me if wrong.